daemon/container.go

```diff
@@ -1235,7 +1235,7 @@ func (container *Container) setupLinkedContainers() ([]string, error) {
 		container.activeLinks = make(map[string]*links.Link, len(children))
 
 		// If we encounter an error make sure that we rollback any network
-		// config and ip table changes
+		// config and iptable changes
```

Comment:
I think it should actually be [`iptables`](https://wiki.archlinux.org/index.php/iptables) (plural) if this refers to the command line utility